### DISCUSSION TITLE
Can parcel be integrated with a external backend?

### DISCUSSION THREAD

**Discussion by yeion7** [parcel-bundler/parcel#8106] [Q&A]
I'm wondering if I can do something like [this](https://vitejs.dev/guide/backend-integration.html) with Parcel

I have a project with django that serve templates and then we load the app, integrate that with vite was quite easy due the built-in manifest, is the same possible with Parcel?

**Comment by lenovouser** [comment]
Currently it is not really possible 😕

There is an [**issue**](https://github.com/parcel-bundler/parcel/issues/4227) regarding this though, but I am not sure when it will be implemented.

**Reply by jtojnar** [reply]
https://github.com/autifyhq/parcel-reporter-bundle-manifest works pretty well for me, this is how I use it: https://github.com/jtojnar/entries/commit/0a0ba47f65baee514eceb4fbb7275cee0fc29ce4

**Reply by lenovouser** [reply]
Yeah, that is a workaround I saw as well, but you have to maintain the `dummy.html` etc.
